go/protocol/keybase1/extras.go

```diff
@@ -1801,3 +1801,8 @@ const (
 	// remote helpers.
 	MDPriorityGit = 32
 )
+
+// IsValid returns true is p is a valid MDPriority, or false otherwise.
+func (p MDPriority) IsValid() bool {
+	return p < 256 && p >= 0
```

Comment:
Is this going to be true even after the `is_placeholder` refactor?

Reply:
Yup it used to be signed `TINYINT`, and not it's unsigned `TINYINT`.

Reply:
But if it's unsigned, then isn't `p >= 0` tautalogical?

Reply:
Yes, but this is typed as `int` generated by msgpack/avdl, which doesn't really support unsigned int. So we need these checks on server side before attempting to put it in to DB.